relayer/src/service.ts

```diff
@@ -105,11 +107,21 @@ export class MessageRelayerService extends BaseServiceV2<
   }
 }
 
+async function returnOnPpidChange(start_ppid: number) {
+  while (start_ppid === ppid) {
+    await sleep(500);
```

Comment:
Nitpick: if you want to do it without a dependency it is trivial `await new Promise(resolve => setTimeout(resolve, 500))` 🤷🏾‍♂️